lib/react_on_rails/configuration.rb

```diff
@@ -123,6 +124,18 @@ def setup_config_values
 
     private
 
+    def check_auto_load_requirements_if_auto_load
+      return if auto_load_bundle != true
```

Comment:
```return unless auto_load_bundle```

Reply:
This is checking the wrong value. 

Should check if `components_subdirectory` `present?`
```
  # components_subdirectory is the name of the subdirectory matched to detect and register components automatically
  # The default is nil. You can enable the feature by updating it in the next line.
  config.components_subdirectory = "ror_components"
```